### ISSUE TITLE
We should not highlight `-Function Get-AllDscEvents` as a function definition

### ISSUE THREAD

**Issue by vors** [SublimeText/PowerShell#80]
``` powershell
Export-ModuleMember -Function Get-AllDscEvents, Trace-xDscOperation, Get-xDscOperation
```

![image](https://cloud.githubusercontent.com/assets/816680/6090230/654e0e14-ae27-11e4-86d7-7cd810091a3c.png)

**Comment by bcrotty** [comment]
Fixed in commit https://github.com/bcrotty/PowerShell/commit/2b96b680314ede3ae7fcb78120b29c0f22f2af95

**Comment by vors** [comment]
Just found that it completely breaks grammar.
@bcrotty did you send it as part of #100 ?

I think it worth to make a release 2.2.0 for that.

**Comment by vors** [comment]
![image](https://cloud.githubusercontent.com/assets/816680/7486471/bc37cb98-f35a-11e4-9393-2f0d1cedfc71.png)

**Comment by bcrotty** [comment]
Using the fix in #100 produces the following:

![2015-05-06 15_38_34-untitled default - sublime text](https://cloud.githubusercontent.com/assets/6155964/7503070/041d3b4a-f406-11e4-93c5-b2d0bd14c530.png)
